src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php

```diff
@@ -159,11 +159,47 @@ private function addUser(string $userId, string $name, string $password): void
             'show_as_rights_template' => '1',
         ]);
 
-        $this->databaseConnection->insert('cms_user_cms_language_mlt', [
-            'source_id' => $userId,
-            'target_id' => $languageEn->id,
-            'entry_sort' => 1,
-        ]);
+        $allLanguages = $this->getAllLanguages();
+        $position = 0;
+        foreach ($allLanguages as $languageId) {
+            $this->databaseConnection->insert('cms_user_cms_language_mlt', [
+                'source_id' => $userId,
+                'target_id' => $languageId,
+                'entry_sort' => ++$position,
+            ]);
+        }
+
+        $allPortals = $this->getAllPortals();
+        $position = 0;
+        foreach ($allPortals as $portalId) {
+            $this->databaseConnection->insert('cms_user_cms_portal_mlt', [
+                'source_id' => $userId,
+                'target_id' => $portalId,
+                'entry_sort' => ++$position,
+            ]);
+        }
+    }
+
+    private function getAllLanguages(): array
+    {
+        $languages = [];
+        $languageList = \TdbCmsLanguageList::GetList();
+        while (false !== $languageObject = $languageList->Next()) {
```

Comment:
Why not loop these above directly?

(As these methods also do not get all languages but all ids.)

Reply:
Yes, that was overly complex - changed.